name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecurityListView.java

```diff
@@ -918,7 +921,7 @@ public String getText(Object element)
         support.addColumn(column);
 
         column = new NoteColumn();
-        column.setEditingSupport(null);
+        new StringEditingSupport(AccountTransaction.class, "note").addListener(this).attachTo(column); //$NON-NLS-1$
```

Comment:
The ```NoteColumn``` already has editing support. If then, it should be activated here, e.g. remove the "null" assignment that deletes the editing.

This code also does not work for me. And it would limit it to AccountTransactions, but there are also many PortfolioTransactions. If at all, it should go against the ```Annotated``` interface. Keep in mind, that for transactions with ```CrossEntry``` *both* notes should be updated.